doc/man7/EVP_KDF-TLS1_PRF.pod

```diff
@@ -47,8 +47,8 @@ this should be more than enough for any normal use of the TLS PRF.
 =item "fips-indicator" (B<OSSL_KDF_PARAM_FIPS_APPROVED_INDICATOR>) <int>
 
 A getter that returns 1 if the operation is FIPS approved, or 0 otherwise.
-This may be used after calling EVP_KDF_derive. It returns 0 if the "ems_check"
-is set to 0 and the "extended master secret" test fails.
+This may be used after calling EVP_KDF_derive. It returns 0 if the related
+check parameter is set to 0.
```

Comment:
It may return 0 if any "***_check" related parameter is set to 0 and the check fails.

Reply:
Same comment applies to other places.

Reply:
Updated.